Test/Graph/test_graph.py

```diff
@@ -173,34 +177,40 @@ def check_alternative(method, alternative):
         check_alternative(lambda: g.atoms_coord, lambda: g.site_offsets)
 
 
-def test_lattice_symmetry():
-    # Check if the symmetry groups make sense
-    for i, graph in enumerate(symmetric_graphs):
-        rots = np.asarray(graph.planar_rotations(rot_symmetry[i]))
-        # R(2pi/period)*R(2pi(1-1/period) = I)
-        assert np.all(rots[-1][rots[1]] == np.arange(graph.n_nodes))
-        # Try a bad rotation and fail
-        with pytest.raises(ValueError):
-            graph.planar_rotations(5)
-
-        # Two reflections returns graph to itself
-        ref = np.asarray(graph.reflections())
-        assert np.all(ref[1][ref[1]] == np.arange(graph.n_nodes))
-
-        # T(0,1)T(0,-1) = I
-        translations = np.asarray(graph.basis_translations())
-        if unit_cells[i] == 8:
-            # [1,0,0] maps to itself
-            assert np.all(translations[1][translations[1]] == np.arange(graph.n_nodes))
-            # [1,1,1] maps to itself
-            assert np.all(
-                translations[-1][translations[-1]] == np.arange(graph.n_nodes)
-            )
-        if unit_cells[i] == 9:
-            # T(2,0)T(1,0) maps to itself
-            assert np.all(translations[2][translations[1]] == np.arange(graph.n_nodes))
-            # T(2,1)T(1,2) maps to itself
-            assert np.all(translations[7][translations[5]] == np.arange(graph.n_nodes))
+@pytest.mark.parametrize("i", range(len(symmetric_graphs)))
```

Comment:
why?

Reply:
Because that ends up in the test name, and `test_graph_thismethod[thislatticename]` is more informative than `test_graph[3]`. When we see a failure in the code it's easier to understand what's going on at a glance instead of having to open the source code and find out what does `[3]` correspond to.

Besides, it's the format I have adopted for all the other tests when I rewrote them. `test_graph` is not uniformed with the rest because that module was not rewritten recently, but I'd like everything to be uniform as we rewrite it..

Reply:
OK, I reverted to being parametrised with number and graph object, which will hopefully improve the names.

Reply:
this will create even more mess because he will use the `__repr__` of the graph objects.
Can't you just do dicts like [here](https://github.com/netket/netket/blob/d240f9c851e17846a5beb29a7a150d7cce1570de/Test/Sampler/test_sampler.py#L23), and using the same keys for the `symmetric_Graphs` dict and the 3 dicts that hold informations?